### DISCUSSION TITLE
About Github

### DISCUSSION THREAD

**Discussion by sevenffxiv14** [community/community#31642] [New to GitHub]
What is Github and how does it help, teach and provide answers

**Comment by toufiqul-hasan** [comment]
Read this:
https://www.simplilearn.com/tutorials/git-tutorial/what-is-github
https://devmountain.com/blog/what-is-github-and-how-do-you-use-it/
https://www.edureka.co/blog/how-to-use-github/

**Comment by jfullstackdev** [comment]
how does it help ? it's for collaboration, for a team to work on their projects, was based on Git, a revision control, because yes, every change is significant in a code, including deletions ! why not Git alone ? back then, there was no centralized space so each team should have their own setup, then some developers realized, why not develop just one central space that can be accessed by anyone ? that is now GitHub

teach ? because GitHub grew bigger and bigger, some tutorials are being hosted here, exactly what I'm doing for my contents being taught in a University, but even projects by others will be educational if that is intended to be, and by digging in some of the actual projects, you learn something

provide answers ? before Discussions feature, there was the Issue tab, if some users of a certain app had the problems, they could raise an issue and discuss that, so it's helping to provide answers, and now that we have this Discussion feature, it's even better, anyone can setup a forum right here on GitHub, without any further special setup, just enable it at the repo settings, even if it's not for a coding project

**Reply by ghost** [reply]
Putting a great short topic

**Comment by Trilochna** [comment]
GitHub is a code hosting platform for version control and collaboration. It lets you and others work together on projects from anywhere. 
The following link will help you learn more about it : 
[https://docs.github.com/en/get-started/quickstart/hello-world](url)

**Comment by Kirtishukla2004** [comment]
hello 
**How Can GitHub Help You?**
GitHub is a social networking company that all the coders, programmers, and engineers use to share their projects with each other.

There are many features on GitHub that give you control over how your work is shown. The contributors of the project have the opportunity to comment on work being done, but they can’t change anything on the website themselves.

If you’re on GitHub, you will also have access to open-source software that others have created. You can also contribute your own efforts to any existing software development projects.
**How does GitHub work?**
GitHub is a social coding platform that allows you to create and publish your projects and help others work together on software or hardware.

GitHub provides the remote hosting of project data while an integrated development environment (IDE) local to the developer’s own machine enables them to make changes in parameters such as source code, scripts and images. Changes can be committed back to GitHub with an automated pull request.

This way, a project can be shared and developed with a larger team all in a central source that is accessible from anywhere.

**Reply by ghost** [reply]
I learned from you
